src/main/java/com/alevel/backend/controller/ExceptionController.java

```diff
@@ -0,0 +1,64 @@
+package com.alevel.backend.controller;
+
+import com.alevel.backend.domain.response.ResponseMessage;
+import com.alevel.backend.domain.response.ResultResponse;
+import com.alevel.backend.domain.response.StatusCode;
+import com.alevel.backend.exception.*;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import javax.naming.LimitExceededException;
+
+@Slf4j
+@RestControllerAdvice
+@RequiredArgsConstructor
+public class ExceptionController {
+
+    @ExceptionHandler(InvalidateUserException.class)
+    @ResponseStatus(HttpStatus.NOT_FOUND)
+    private ResultResponse InvalidateUserException(InvalidateUserException e) {
+        log.info(e.getMessage());
+        e.printStackTrace();
+        return ResultResponse.fail(StatusCode.NOT_FOUND, ResponseMessage.INVALIDATED_USER);
+    }
+
+    @ExceptionHandler(WithdrawnUserException.class)
+    private ResultResponse WithdrawnUserException(WithdrawnUserException e) {
+        log.info(e.getMessage());
+        e.printStackTrace();
+        return ResultResponse.fail(StatusCode.NOT_FOUND, ResponseMessage.WITHDRAWN_USER);
+    }
+
+    @ExceptionHandler(InvalidatePasswordException.class)
+    private ResultResponse InvalidatePasswordException(InvalidatePasswordException e) {
+        log.info(e.getMessage());
+        e.printStackTrace();
+        return ResultResponse.fail(StatusCode.UNAUTHORIZED, ResponseMessage.INVALIDATED_PASSWORD);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)